src/th/niceoppai/build.gradle

```
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    extName = 'Niceoppai'
+    pkgNameSuffix = 'th.niceoppai'
+    extClass = '.Niceoppai'
+    extVersionCode = 1
```

```suggestion
    extVersionCode = 25
```
... to continue from the Madara version

---

It's the same package name and classname so it should appear as an update. Please apply the changes suggested by Vetle.

---

@alessandrojean @vetleledaal  thank you. it applied.